### PR TITLE
Chore/0.7.11

### PR DESCRIPTION
**Pull request by guidiaz**

### CODE REVIEW

File: migrations/scripts/3_WitnetBytecodes.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ module.exports = async function (deployer, network, [, from]) {
         // deploy instance only if not found in current network:
         utils.traceHeader("Singleton inception of 'WitnetBytecodes':")
         const balance = await web3.eth.getBalance(from)
-        const gas = singletons.WitnetBytecodes.gas
+        const gas = singletons.WitnetBytecodes.gas || 5000000
```
Comment (aesedepece) [inline migrations/scripts/3_WitnetBytecodes.js:52]: Can we get the 5000000 declared as a constant somewhere and import upon usage?

File: package.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
     "solidity-coverage": "0.7.16",
     "truffle": "~5.8.2",
     "truffle-assertions": "0.9.2",
-    "truffle-flattener": "~1.6.0",
+    "truffle-flattener": "",
```
Comment (aesedepece) [inline package.json:65]: No version?

**Review by aesedepece — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (guidiaz) [reply package.json:65]: Oops !

File: migrations/scripts/3_WitnetBytecodes.js (hunk 1 above)
Comment (guidiaz) [reply migrations/scripts/3_WitnetBytecodes.js:52]: Typo, it should be removed

**Review by aesedepece — APPROVED** [review]

**Pull request closed without merging**